project_resource_calendar/tests/common.py

```diff
@@ -8,7 +8,12 @@ class TestCalendarEventCommon(common.TransactionCase):
 
     def setUp(self):
         super(TestCalendarEventCommon, self).setUp()
-
+        # Initialization of param values
+        room_name = 'Testing_Room_type_2'
```

Comment:
this variable is used only one time you can directly put the room name in the creation of the resource.calendar.room object

Reply:
@rimbendhaou Done